python/plugins/processing/algs/gdal/ClipRasterByMask.py

```diff
@@ -81,9 +94,26 @@ def initAlgorithm(self, config=None):
                                                         self.tr('Match the extent of the clipped raster to the extent of the mask layer'),
                                                         defaultValue=True))
         self.addParameter(QgsProcessingParameterBoolean(self.KEEP_RESOLUTION,
-                                                        self.tr('Keep resolution of output raster'),
+                                                        self.tr('Keep resolution of input raster'),
```

Comment:
Is this a bug fix? Would be good to separate this commit so it can be backported